From 04271eb2d7fcc5cf8565187b06c7d601134bb955 Mon Sep 17 00:00:00 2001
From: Changhyeok Bae <changhyeok.bae@gmail.com>
Date: Thu, 6 Jul 2017 09:41:31 +0900
Subject: [PATCH] Disable eglfs-brcm explicitly

:Release Notes:
Disable eglfs-brcm explicitly

:Detailed Notes:
After applying userland for omxplayer, eglfs-brcm is built.
However we don't need eglfs-brcm because webOS uses drm and eglfs for
qpa.

:Testing Performed:
Only build tested.

:QA Notes:

:Issues Addressed:
[PLAT-42918] qtbase build failures on rpi3 no shared state build

Change-Id: I9392e5aec929fe7d1c3dc9f52f94323ede3f86e0
---
 src/gui/configure.json                           | 16 +---------------
 .../deviceintegration/deviceintegration.pro      |  1 -
 2 files changed, 1 insertion(+), 16 deletions(-)

diff --git a/src/gui/configure.json b/src/gui/configure.json
index 8fbc24b0dd..64d1786f8d 100644
--- a/src/gui/configure.json
+++ b/src/gui/configure.json
@@ -864,15 +864,6 @@
             },
             "use": "egl xlib"
         },
-        "egl-brcm": {
-            "label": "Broadcom EGL (Raspberry Pi)",
-            "type": "compile",
-            "test": {
-                "include": [ "EGL/egl.h", "bcm_host.h" ],
-                "main": "vc_dispmanx_display_open(0);"
-            },
-            "use": "egl bcm_host"
-        },
         "egl-egldevice": {
             "label": "EGLDevice",
             "type": "compile",
@@ -1400,11 +1391,6 @@
             "condition": "!config.android && !config.darwin && !config.win32 && features.egl",
             "output": [ "privateFeature" ]
         },
-        "eglfs_brcm": {
-            "label": "EGLFS Raspberry Pi",
-            "condition": "features.eglfs && tests.egl-brcm",
-            "output": [ "privateFeature" ]
-        },
         "eglfs_egldevice": {
             "label": "EGLFS EGLDevice",
             "condition": "features.eglfs && tests.egl-egldevice && features.kms",
@@ -1947,7 +1933,7 @@ QMAKE_LIBDIR_OPENGL[_ES2] and QMAKE_LIBS_OPENGL[_ES2] in the mkspec for your pla
                     "section": "EGLFS details",
                     "condition": "features.eglfs",
                     "entries": [
-                        "eglfs_openwfd", "eglfs_viv", "eglfs_viv_wl", "eglfs_rcar", "eglfs_egldevice", "eglfs_gbm", "eglfs_vsp2", "eglfs_mali", "eglfs_brcm", "eglfs_x11"
+                        "eglfs_openwfd", "eglfs_viv", "eglfs_viv_wl", "eglfs_rcar", "eglfs_egldevice", "eglfs_gbm", "eglfs_vsp2", "eglfs_mali", "eglfs_x11"
                     ]
                 },
                 "linuxfb", "vnc", "mirclient",
diff --git a/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro b/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro
index 360536d22f..3c9fcce7da 100644
--- a/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro
+++ b/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro
@@ -5,7 +5,6 @@ qtConfig(eglfs_x11): SUBDIRS += eglfs_x11
 qtConfig(eglfs_gbm): SUBDIRS *= eglfs_kms_support eglfs_kms
 qtConfig(eglfs_egldevice): SUBDIRS *= eglfs_kms_support eglfs_kms_egldevice
 qtConfig(eglfs_vsp2): SUBDIRS += eglfs_kms_vsp2
-qtConfig(eglfs_brcm): SUBDIRS += eglfs_brcm
 qtConfig(eglfs_mali): SUBDIRS += eglfs_mali
 qtConfig(eglfs_viv): SUBDIRS += eglfs_viv
 qtConfig(eglfs_rcar): SUBDIRS += eglfs_rcar
